CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## 0.14.0-rc1 / 2015-05-21
```

Comment:
Why actually? I'm used from other projects that the changelog presents the current state. It could be `0.14.0 / unreleased` and changed as soon as it gets release. At the moment the changelog doesn't help to understand what changes are available in 0.1.4.0rc1